examples/reference/containers/plotly/DynamicMap.ipynb

```diff
@@ -74,6 +75,41 @@
     "<img src='https://s3-eu-west-1.amazonaws.com/assets.holoviews.org/gifs/examples/containers/plotly/DynamicMap.gif'>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This example can also be displayed using [Plotly Dash](https://dash.plotly.com/):\n",
+    "\n",
+    "```python\n",
+    "from holoviews.plotting.plotly.dash import holoviews_to_dash\n",
+    "import dash\n",
+    "import dash_html_components as html\n",
+    "\n",
+    "# Create App\n",
+    "external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']\n",
```

Comment:
It would be good to avoid depending on code in anyone's personal accounts; does it really need the .css decoration? Or could the file be moved somewhere more permanent?

Reply:
Ahh, yeah, I'll pull that out if we keep the examples in these notebooks.